x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -75,6 +75,29 @@ create a notification through SQS. Please see
 https://docs.aws.amazon.com/AmazonS3/latest/dev/ways-to-add-notification-config-to-bucket.html#step1-create-sqs-queue-for-notification[create-sqs-queue-for-notification]
 for more details.
 
+[float]
+=== Parallel Processing
+Running multiple Filebeat instances in parallel to read from the same SQS queue
+is available for all users without any extra configuration for s3 input. This
+feature helps users to horizontally scale the processing when there are large
+amount of logs flowing into S3 bucket which leads to large amount of messages
+going through the queue.
```

Comment:
The first sentence in this paragraph sounds a bit odd to me, what do you think about rephrasing it to something like this:
```
Multiple Filebeat instances can read from the same SQS queues at the same time. This
feature helps users to horizontally scale the processing when there are large
amount of logs flowing into S3 bucket which leads to large amount of messages
going through the queue. No additional configuration is needed for this.
```

Reply:
A lot better 😂 Thank you!!!